### ISSUE TITLE
Clarification on the totalPending api stat

### ISSUE THREAD

**Issue by calebcall** [coinfoundry/miningcore#437]
Is this stat the total amount owed to a miner that has fully confirmed and not yet paid out?  For some reason I was thinking this was the total found that hadn't paid (pending blocks included) but that was usually at 0 even with blocks found but still pending.

**Comment by calebcall** [comment]
Sorry, pendingBalance (at the /api/pools/{poolid}/miners/{miner-wallet-address} endpoint).  Got it confused with totalPaid.

**Comment by oliverw** [comment]
pendingBalance is the amount the pool owes the account (miner) below the pool's minimum payout.

**Comment by calebcall** [comment]
but those are blocks that have confirmed, right?  (I guess that would make sense since until the block fully confirms, it's not really"found")

**Comment by tron1103** [comment]
So when I have a lot of pending LTC shares, e.g.
```
pendingShares | 3536842149
pendingBalance | 0
```
or BTC shares, e.g.
```
pendingShares | 961651340
pendingBalance | 0
```
I need a block to be found to get the pendingBalance for them, right? Is there another way to force updates via PostGres as we have not found an LTC nor BTC block yet :(
Btw, current v45 running with the following payout settings for the LTC as well as the BTC pool with just one miner each:
```
 "paymentProcessing": {
        "enabled": true,
        "minimumPayment": 0.01,
        "payoutScheme": "PPLNS",
        "payoutSchemeConfig": {
          "factor": 0.1
        }
```

**Comment by oliverw** [comment]
@tron1103 Correct, with PPLNS shares have no monetary worth. A PPS pool in contrast pays per share (hence the name) and shoulders the full risk of a long round.
